tools/tags/tags.js

```diff
@@ -11,14 +16,48 @@ async function getAemRepo(project, opts) {
   return null;
 }
 
+async function getTags(path, opts) {
+  const activeTag = path.split('cq:tags').pop().replace('.1.json', '').slice(1);
+  const resp = await fetch(path, opts);
+  if (!resp.ok) return null;
+  const json = await resp.json();
+  const tags = Object.keys(json).reduce((acc, key) => {
+    if (json[key]['jcr:primaryType'] === 'cq:Tag') {
+      acc.push({
+        path: `${path.replace(TAG_EXT, '')}/${key}${TAG_EXT}`,
+        activeTag,
+        name: key,
+        title: json[key]['jcr:title'] || key,
+        details: json[key],
+      });
+    }
+    return acc;
+  }, []);
+
+  return tags;
+}
+
 (async function init() {
   const { context, actions, token } = await DA_SDK;
   if (!context || !token) return;
   const opts = { headers: { Authorization: `Bearer ${token}` } };
   const aemRepo = await getAemRepo(context, opts);
   if (!aemRepo) return;
+
+  const rootTags = await getTags(`https://${aemRepo}${ROOT_TAG_PATH}${TAG_EXT}`, opts);
+
+  if (!rootTags || rootTags.length === 0) {
+    const login = document.createElement('a');
+    login.textContent = 'Please login to AEM to view tags';
```

Comment:
Nit: Login is a noun
```suggestion
    login.textContent = 'Please log in to AEM to view tags';
```

Reply:
Thanks!